drop-storage/src/lib.rs

```diff
@@ -483,6 +494,101 @@ impl Storage {
         Ok(())
     }
 
+    pub async fn transfers_to_retry(&self) -> Result<Vec<TransferToRetry>> {
+        let mut conn = self.conn.begin().await?;
+
+        let rec_transfers = sqlx::query!(
+            r#"
+            SELECT transfers.id as  "id: Hyphenated", peer, COUNT(transfer_cancel_states.id) as cancel_count
```

Comment:
Because sqlx expects `uuid::Uuid` as binary and fails if it's stored as hex string. `uuid::Hypenated` is a way to tell to sqlx that it is stored as a string